packages/design-system/src/components/Table/TableCell.jsx

```diff
@@ -0,0 +1,84 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+export const TableCell = ({
+  children,
+  className,
+  data,
+  stackedTitle,
+  stackedClassName,
+  type,
+  ...attributeOptions
+}) => {
+  const classes = classNames(
+    'ds-c-table__cell',
+    className,
+    type === 'numeric' ? 'ds-c-table__cell--numeric' : null
+  );
+
+  const stackedClasses = classNames(
```

Comment:
What do you think about moving this definition into `renderStackedTitle`, as thats the only place these are used?
Something like this:
```
const renderStackedTitle = () => {
   if (stackedTitle && stackedTitle.length > 0) {
       const stackedClasses = classNames(...);
       return (
           <span aria-hidden="true" className={stackedClasses}>...</span>
       )
   }
}